### PR TITLE
Added miscellaneous folder and electricity bill generator

### PR DESCRIPTION
**Pull request by Unishkhadka**
## New Pull Request

- [x] Descriptive names
- [x] Well-Commented Code
- [x] Descriptive Commit
- [x] Well Documented readme.md
- [x] Format for pull request [type]: Description

### CODE REVIEW

File: Basic Questions/10_prime_numbers.c
Comment (razzat008) [inline Basic Questions/10_prime_numbers.c:1]: Add output for question number 10 in basic questions directory

Comment (Unishkhadka) [reply Basic Questions/10_prime_numbers.c:1]: Updated! Thank you.

**Review by razzat008 — APPROVED** [review]
great!

File: test.txt
Comment (gauravfs-14) [inline test.txt:1]: Please remove the unwanted test file before merging.

Comment (Unishkhadka) [reply test.txt:1]: Sorry! Now, i think it should be fine. Removed!

**Review by gauravfs-14 — APPROVED** [review]
Don't create unwanted commits the next time like adding and removing an unwanted file. like 'html.html'.

**Pull request merged**